index.d.ts

```diff
@@ -113,10 +117,10 @@ export const animated: {
   [Tag in keyof JSX.IntrinsicElements]: ComponentClass<
     JSX.IntrinsicElements[Tag]
   >
-}
+};
 
-type TransitionKeyProps = string | number
-type TransitionItemProps = string | number | object
+type TransitionKeyProps = string | number;
+type TransitionItemProps = string | number | object;
```

Comment:
Intentionally added lonely semi-colons? 😊

Reply:
Oops; My linter is having a fit with the code styles and might have accidentally added them. Personally I think the standard should be semi-colons, but I thought it was better to be consistent

Reply:
Yeah I agree @PrecociouslyDigital. Cool!